src/main/java/org/broadinstitute/hellbender/engine/LocusWalker.java

```diff
@@ -39,7 +39,7 @@
  */
 public abstract class LocusWalker extends GATKTool {
 
-    @Argument(fullName = "maxDepthPerSample", shortName = "maxDepthPerSample", doc = "Maximum number of reads to retain per sample per locus. Reads above this threshold will be downsampled. Set to 0 to disable.", optional = true)
+    @Argument(fullName = "max-depth-per-sample", shortName = "max-depth-per-sample", doc = "Maximum number of reads to retain per sample per locus. Reads above this threshold will be downsampled. Set to 0 to disable.", optional = true)
```

Comment:
There are two error messages that refer to the old `maxDepthPerSample` one in LocusWalker.getDownsamplingInfo() and the other in LocusWalkerSpark.  Could you extract a constant from this and have all three locations use it?

Reply:
done